index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>| Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="mobile-main">
+        <div class="mobile-header">
+            <div class="app-bar">
+                <div class="logo">
+                    My Logo
+                </div>
+                <div class="menu">
+                    <img src="/Union1.png" alt="">
+                </div>
+            </div>
+            <div class="header-main-content">
+                <div class="introduction-primary-text">
+                    I’m Andile
+                    Glad to see you!
+                </div>
+
+                <div class="introduction-supporting-text">
+                    I’m a software developer! I 
+                    can help you build a product 
+                    , feature or website Look 
+                    through some of my work and 
+                    experience! If you like what 
+                    you see and have a project you 
+                    need coded, don’t hestiate 
+                    to contact me.
+                </div>
```

Comment:
Awesome paragraph 💯 but let's see... We could decide to have the paragraph here as a `p` tag.
 * This way will be semantically correct and the screen readers and other technologies will thank you. 🙏